modules/digging-into-machine-config.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/introduction-openshift-architecture.adoc
+
+[id="digging-into-machine-config_{context}"]
+= Digging into machine configuration
+
+Machine Config Pools manage a cluster of nodes and their corresponding Machine Configs. Machine Configs contain configuration information for a cluster. To list all Machine Config Pools that are known:
+
+----
+$ oc get machineconfigpools
+
+NAME   CONFIG                                  UPDATED UPDATING DEGRADED
+master master-1638c1aea398413bb918e76632f20799 False   False    False
+worker worker-2feef4f8288936489a5a832ca8efe953 False   False    False
+----
+
+To list all Machine Configs:
+
+----
+$ oc get machineconfig
+
+NAME                                      GENERATEDBYCONTROLLER   IGNITIONVERSION   CREATED   OSIMAGEURL
+
+00-master                                 4.0.0-0.150.0.0-dirty   2.2.0             16m
+00-master-ssh                             4.0.0-0.150.0.0-dirty                     16m
+00-worker                                 4.0.0-0.150.0.0-dirty   2.2.0             16m
+00-worker-ssh                             4.0.0-0.150.0.0-dirty                     16m
+01-master-kubelet                         4.0.0-0.150.0.0-dirty   2.2.0             16m
+01-worker-kubelet                         4.0.0-0.150.0.0-dirty   2.2.0             16m
+master-1638c1aea398413bb918e76632f20799   4.0.0-0.150.0.0-dirty   2.2.0             16m
+worker-2feef4f8288936489a5a832ca8efe953   4.0.0-0.150.0.0-dirty   2.2.0             16m
+----
+
+The Machine Config Operator acts somewhat differently than Ignition when it comes to applying these machineconfigs. The machineconfigs are read in order (from 00* to 99*). Labels inside the machineconfigs identify the type of node each is for (master or worker). If the same file appears in multiple machineconfig files, the last one wins. So, for example, any file that appears in a 99* file would replace the same file that appeared in a 00* file. The input machineconfig objects are unioned into a "rendered" machineconfig object, which will be used as a target by the operator and is the value you can see in the machineconfigpool.
```

Comment:
This is the most important part of this file. Plz move it to a different module.